test/hotspot/jtreg/vmTestbase/nsk/share/Log.java

```diff
@@ -406,10 +337,9 @@ private void logExceptionForFailureAnalysis(String msg) {
     /////////////////////////////////////////////////////////////////
 
     /**
-     * Redirect log to the given <code>stream</code>, and switch
-     * log mode to verbose.
+     * Redirect log to the given <code>stream</code>
      * Prints errors summary to current stream, cancel current stream
-     * and switches to new stream. Turns on verbose mode for new stream.
+     * and switches to new stream.
```

Comment:
Does it really do all this? It looks to me like it just switches to the new stream. I'm not sure what is meant by "error summary" and cancelling.

Reply:
Ok, but unless it is going to be removed by another CR soon, I think the comments should at least reflect what it currently does.